pos_sale_order_delivery_carrier/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "POS Sale Order Delivery Carrier",
```

Comment:
I think this module shouldb e named `pos_order_to_sale_order_delivery`, because it seems to be a glue module between `pos_order_to_sale_order` and the core `delivery` module.